@here/harp-examples/src/rendering_globe-atmosphere.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EarthConstants, GeoCoordinates, sphereProjection } from "@here/harp-geoutils";
+import { MapControls, MapControlsUI } from "@here/harp-map-controls";
+import {
+    AtmosphereLightMode,
+    CopyrightElementHandler,
+    MapView,
+    MapViewAtmosphere
+} from "@here/harp-mapview";
+import { InterpolatedClipPlanesEvaluator } from "@here/harp-mapview/lib/ClipPlanesEvaluator";
+import { APIFormat, AuthenticationMethod, OmvDataSource } from "@here/harp-omv-datasource";
+import { WebTileDataSource } from "@here/harp-webtile-datasource";
+import { apikey, copyrightInfo } from "../config";
+
+export namespace GlobeAtmosphereExample {
+    enum DataSourceVariant {
+        Omv,
+        Satellite
+    }
+
+    // Create a new MapView for the HTMLCanvasElement of the given id.
+    function initializeMapView(id: string): MapView {
+        const canvas = document.getElementById(id) as HTMLCanvasElement;
+        // tslint:disable-next-line:deprecation
+        const clipPlanesEvaluator = new InterpolatedClipPlanesEvaluator(
```

Comment:
Is it possible to make this effect work with our other ClipPlanesEvaluators? Maybe we need to use a different camera(with different near/far plane setup) to render the atmospheric geometries.

Reply:
Yep this final plan is to integrate InterpolatedClipPlanesEvaluator into the MapViewAtmosphere, so near/far it is gonna be changed dynamically within onBeforeRender(), and restored onAfterRender(). This way all tiles geometry will use TiltViewClipPlanes... while the atmosphere geometry will have its own.

Reply:
I have resolved this problem so there is need to define _special_ clip planes evaluator for MapView component. Instead of creating new camera setup internally, we just update camera near/far just before rendering atmosphere object (sky), after render clip planes are restored for the rest of scene, so full integration is now added.